### PR TITLE
Miscellaneous cleanups

### PR DESCRIPTION
**Pull request by nedbat**
I'm thinking about how to make this repo easier to use for tossed-off programs to support Actions, etc.

Along the way, I found these things to clean up.

### CODE REVIEW

File: edx_repo_tools/dev/show_hooks.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from git.repo.base import Repo
 
 from edx_repo_tools.auth import pass_github
-from helpers import paginated_get
+from edx_repo_tools.helpers import paginated_get
```
Comment (timmc-edx) [inline edx_repo_tools/dev/show_hooks.py:10]: I'm not following this one -- where is this module declared, if not at the top level?

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 
 help:				## display this help message
 	@echo "Please use \`make <target>' where <target> is one of"
 	@awk -F ':.*?## ' '/^[a-zA-Z]/ && NF==2 {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST) | sort
 
+clean:				## remove transient artifacts
+	rm -rf .*cache *.egg-info .coverage build/ htmlcov/
+	rm -rf .oep2-workspace/
+	rm -rf __pycache__ */__pycache__ */*/__pycache__
+	rm -rf *.pyc */*.pyc */*/*.pyc
```
Comment (timmc-edx) [inline Makefile:11]: If possible, I'd like to use what's in the cookiecutter, which will go to all levels:

https://github.com/openedx/edx-cookiecutters/blob/f49b8beb757ae250747dea191e57e6ce037d9686/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/Makefile#L14

File: edx_repo_tools/dev/show_hooks.py (hunk 1 above)
Comment (nedbat) [reply edx_repo_tools/dev/show_hooks.py:10]: It's in the edx_repo_tools directory now, it had been at the top of the repo, which wouldn't work when installed.

File: Makefile (hunk 2 above)
Comment (timmc-edx) [reply Makefile:11]: (or just in general, take what's in the cookiecutter and then add on anything else we need to remove, like this `.oep2-workspace` thing)

Comment (nedbat) [reply Makefile:11]: I knew there was a better way to do it....

File: edx_repo_tools/dev/show_hooks.py (hunk 1 above)
Comment (timmc-edx) [reply edx_repo_tools/dev/show_hooks.py:10]: Oh, I see the rename now. :-)

**Review by timmc-edx — APPROVED** [review]

**Pull request merged**